src/cpu.c

```diff
@@ -3,6 +3,9 @@
 #include "rom.h"
 #include "cpu.h"
 
+#define NIBBLE_MASK (0xF)
```

Comment:
this only covers the lower nibble, doesn't it?
As 0x0F != 0xF0

Reply:
Absolutely, that should be named LNIBBLE_MASK or LOWER_NIBBLE_MASK?